frontend/src/host_orchestrator/orchestrator/cvd/cvd.go

```diff
@@ -122,6 +124,50 @@ func (c *Command) Run() error {
 	return nil
 }
 
+// Same as `Run()`, but error won't be populated in case of failed executions, the caller gets the exit code value.
+func (c *Command) Execute() (int, error) {
+	// Makes sure cvd server daemon is running before executing the cvd command.
```

Comment:
why? The server should start on its own automatically

Reply:
we should add the explanation in this line instead.

https://github.com/google/android-cuttlefish/blob/main/frontend/src/host_orchestrator/orchestrator/cvd/cvd.go#L127-L132